Source/Tests/LinuxSyscalls/x32/Memory.cpp

```diff
@@ -26,18 +26,26 @@ static std::string get_fdpath(int fd)
 
 namespace FEX::HLE::x32 {
   void RegisterMemory() {
-    REGISTER_SYSCALL_IMPL_X32(mmap, [](FEXCore::Core::CpuStateFrame *Frame, uint32_t addr, uint32_t length, int prot, int flags, int fd, int32_t offset) -> uint64_t {
+    struct old_mmap_struct {
+      uint32_t addr;
+      uint32_t len;
+      uint32_t prot;
+      uint32_t flags;
+      uint32_t fd;
+      uint32_t offset;
+    };
+    REGISTER_SYSCALL_IMPL_X32(mmap, [](FEXCore::Core::CpuStateFrame *Frame, old_mmap_struct *const arg) -> uint64_t {
```

Comment:
Shouldn't this be `const old_mmap_struct * arg` ? (const data, not const pointer?)

Reply:
Oops yes, fixed.